assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
+        focusId = ("" + $this.attr("href")).trim();
```

Comment:
> What issues are you referring to?

I was referring to the `trim` and the `indexOf` felt overkill because you are using a fragment inside of `$()` but @rpowis pointed out to me that this is the actual in browser functionality:
- click url with fragment
- browser goes to element on page.